math/genvector/inc/Math/GenVector/Transform3D.h

```diff
@@ -606,36 +868,96 @@ class Transform3D {
    /**
       make transformation from first a rotation then a translation
    */
-   void  AssignFrom( const Rotation3D & r, const Vector & v);
+   void AssignFrom( const Rotation3D & r, const Vector & v)
+   {
+     // assignment  from rotation + translation
+     
+     T rotData[9];
+     r.GetComponents(rotData, rotData +9);
+     // first raw
+     for (int i = 0; i < 3; ++i)
+       fM[i] = rotData[i];
+     // second raw
+     for (int i = 0; i < 3; ++i)
+       fM[kYX+i] = rotData[3+i];
+     // third raw
+     for (int i = 0; i < 3; ++i)
+       fM[kZX+i] = rotData[6+i];
+     
+     // translation data
+     T vecData[3];
+     v.GetCoordinates(vecData, vecData+3);
+     fM[kDX] = vecData[0];
+     fM[kDY] = vecData[1];
+     fM[kDZ] = vecData[2];
+   }
+
 
    /**
       make transformation from only rotations (zero translation)
    */
-   void  AssignFrom( const Rotation3D & r);
+   void  AssignFrom( const Rotation3D & r)
```

Comment:
The last line gives me

```
Oberon ~/Projects/ROOT/source > ./git-clang-format --commit 22a3099132 --diff
fatal: bad revision '22a3099132'
```

Looking at git log I figure the correct revision to compare to is

```
> ./git-clang-format --commit 29722bb7dfd32350c0e5d31c654c5df32633d1cc --diff
```

which gives me a much more reasonable looking diff.